lib/kino/widget.ex

```diff
@@ -1,30 +1,21 @@
 defmodule Kino.Widget do
-  @moduledoc """
-  A structure representing a widget process.
-  """
-
-  defstruct [:pid, :type]
-
-  @type t :: %__MODULE__{
-          pid: pid(),
-          type: atom()
-        }
+  @moduledoc false
 
   @doc false
-  @spec start!(module(), keyword(), atom()) :: Kino.Widget.t()
-  def start!(module, opts, type) do
+  @spec start!(module(), keyword()) :: pid()
+  def start!(module, opts) do
     case DynamicSupervisor.start_child(Kino.WidgetSupervisor, {module, opts}) do
```

Comment:
I would call this module for now. You might as well call `DynamicSupervisor.start_child(Kino.WidgetSupervisor, {module, opts})` directly in `Kino.VegaLite` (and you know it can only succeed with `{:ok, pid}` anyway).

Reply:
You mean it should be fine to start and aggressively match against `{:ok, pid}` in `Kino.VegaLite`?

Reply:
Yes. Do you know any reason why it should fail?

Reply:
I don't :) And in cases like not enough memory it doesn't really matter since things are bad enough already, right?